follow/migrations/0007_remoterequest_from_user_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2022-04-03 23:00
```

Comment:
I'm not a migration expert, but I imagine @armiantos would say to delete these and run `makemigrations` again so that there is one migration that gives us the changes we want

Reply:
Yea let's try to keep it clean, this isn't hard to do